examples/from_source/WORKSPACE

```diff
@@ -0,0 +1,57 @@
+workspace(
+    name = "from_source",
+    managed_directories = {"@npm": ["node_modules"]},
+)
+
+load("@bazel_tools//tools/build_defs/repo:git.bzl", "git_repository")
+
+git_repository(
+    name = "build_bazel_rules_nodejs",
```

Comment:
I think in this case it should just be `local_repository(path="../..")` maybe? That's what we do in a couple other cases of nested workspaces